modules/ROOT/pages/cloudhub-fabric.adoc

```diff
@@ -129,15 +129,31 @@ Number of Workers: 1
 |===
 
 
-== Persistent Queuing Behavior for Applications Containing Batch Jobs
+== Persistent Queues for Applications Containing Batch Jobs
 
-When you deploy an application containing batch jobs to CloudHub with persistent queues enabled, the batch jobs use CloudHub's persistent queuing feature for the batch queuing functionality.
+When you deploy an application containing batch jobs to CloudHub with persistent queues enabled, the batch jobs use CloudHub persistent queuing for the batch queuing functionality.
 
-When using persistent queues, keep in mind the following limitations:
+If you enable persistent queues, keep in mind the following limitations:
```

Comment:
Might replace "keep in mind" with a non-idiom for translation purposes - maybe "note the following limitations"?